ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -118,6 +118,7 @@ const triggerSet: TriggerSet<Data> = {
         text: {
           en: 'Drop Puddle Outside',
           de: 'Lege Fläche außen ab',
+          cn: '去场边放火圈',
```

Comment:
```suggestion
          cn: '去场边放毒圈',
```
This is actually a vemon puddle, and vemon give a impression of remain on the ground.
Fire mainly give a impression of quick and instant.